libs/megaparse/tests/test_parsers.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import pytest
+from megaparse.parser.doctr_parser import DoctrParser
+from megaparse.parser.llama import LlamaParser
+from megaparse.parser.megaparse_vision import MegaParseVision
+from megaparse.parser.unstructured_parser import UnstructuredParser
+from megaparse_sdk.schema.extensions import FileExtension
+
+PARSER_LIST = [
+    UnstructuredParser,
+    DoctrParser,
+]
+
+
+@pytest.mark.parametrize("parser", PARSER_LIST)
+@pytest.mark.parametrize("extension", list(FileExtension))
+def test_sync_parser(parser, extension):
+    directory = "./tests/supported_docs"
+    file_path = next(
+        (
+            os.path.join(root, file)
+            for root, _, files in os.walk(directory)
+            for file in files
+            if file.endswith(extension.value)
```

Comment:
We can move filter of parser supported extensions here

Reply:
I am not sure to understand

Reply:
I also want to verify it throws an error when the specific parser does not support this extension, for example : Unstructured supports every possible extensions in MegaParse but doctr only supports PDF for the moment